app/components/shared/events.tsx

```diff
@@ -9,19 +9,19 @@ export function EventsList({ events }: { events: Event[] }) {
 	}
 
 	return (
-		<ul className="space-y-4">
+		<ul className="grid gap-6 md:grid-cols-2 lg:grid-cols-3">
```

Comment:
![image](https://github.com/bandungdevcom/bandungdev.com/assets/110674466/192c1a82-81f3-4c77-a657-333737b9babd)

The card on the landing page looks broken, u can change the section size on the landing page or change the grid size to two on the landing page.

Reply:
oke, I've fix it